flight/test_files/state_machine_test.py

```diff
@@ -15,7 +15,7 @@ async def run_test(_sim: bool) -> None:  # Temporary fix for unused variable
     _sim : bool
         Whether to run the state machine in simulation mode.
     """
-    FlightManager().start_manager()
+    FlightManager().start_manager(_sim)
```

Comment:
Change the argument in this function to be called `sim` instead of `_sim`, since it is actually implemented now.